src/journal/journald-kmsg.c

```diff
@@ -436,9 +437,9 @@ int server_open_kernel_seqnum(Server *s) {
                 return 0;
         }
 
-        r = posix_fallocate(fd, 0, sizeof(uint64_t));
+        r = posix_fallocate_loop(fd, 0, sizeof(uint64_t));
         if (r != 0) {
```

Comment:
since this returns negative errno values now, like our usual coding style, let's handle like we handle them everywhere else too, i.e. with an `if (r < 0)` check